tests/typecheck/test_settings.yml

```diff
@@ -12,7 +12,7 @@
         from base import *
         SECRET_KEY = 112233
         NUMBERS = ['one', 'two']
-        DICT = {}  # type: ignore
+        DICT = {}  # type: ignore[var-annotated]
```

Comment:
Oh yeah we don't have ruff checks in these yml tests sadly :/ 
It should have been caught by https://docs.astral.sh/ruff/rules/blanket-type-ignore/